test/parallel_api/numeric/numeric.ops/inclusive_scan_by_segment.pass.cpp

```diff
@@ -27,6 +27,8 @@ using namespace oneapi::dpl::execution;
 #endif
 using namespace TestUtils;
 
+//#define DUMP_CHECK_RESULTS
+
 struct test_inclusive_scan_by_segment
 {
     // TODO: replace data generation with random data and update check to compare result to
```

Comment:
Yes, we need to use srand/rand to exclude any assumptions on checking result stage.

Reply:
Implemented in separate branch